keyboards/ergodox/infinity/rules.mk

```diff
@@ -62,15 +62,10 @@ CUSTOM_MATRIX ?= yes # Custom matrix file
 SERIAL_LINK_ENABLE = yes
 VISUALIZER_ENABLE ?= yes
 LCD_ENABLE ?= yes
-LED_ENABLE ?= no
+BACKLIGHT_ENABLE ?= yes
 LCD_BACKLIGHT_ENABLE ?= yes
 MIDI_ENABLE = no
 RGBLIGHT_ENABLE = no
 
-ifdef LCD_ENABLE
 include $(SUBPROJECT_PATH)/drivers/gdisp/st7565ergodox/driver.mk
-endif
-
-ifdef LED_ENABLE
-include $(SUBPROJECT_PATH)/drivers/gdisp/IS31FL3731C/driver.mk
-endif
\ No newline at end of file
+include $(SUBPROJECT_PATH)/drivers/gdisp/IS31FL3731C/driver.mk
```

Comment:
Both drivers have to be compiled in, since we can't pass custom parameters to the uGFX configuration. uGFX is only compiled once per keyboard, and not per keymap, so switching behavior per keymap wouldn't work. Memory is not a problem, at least not yet for the Infinity, so this shouldn't matter.